.github/workflows/master_build_pr.yml

```diff
@@ -35,6 +42,70 @@ jobs:
             echo "$FILE"
             echo "Starting Docker"
             docker build -f $FILE -t $IMAGE_NAME .
+            imageScan $IMAGE_NAME
+            set +e
+          }
+          imageScan(){
+            set -e
+            IMAGE_NAME="$1"
+            trivy image -f json -o trivy_image_scan_result.json  --severity HIGH,CRITICAL --scanners vuln $IMAGE_NAME:latest
+            dockle -f json -o dockle_image_scan_result.json $IMAGE_NAME:latest
+            python3 - <<EOF
```

Comment:
You are  Passed - You can define a python method  inside a anonymous python block -> which is inside a bash function call - which inturn is inside a bash script -> which is called via the nodejs (a.k.a Github Actions).

Now split this so humans can understand it.